### PR TITLE
Update wds-boot-support.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document to represent  OS deployed vs boot image version as per the table

Per issue#https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10973

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: windows/deployment/wds-boot-support.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ When you PXE-boot from a WDS server that uses the **boot.wim** file from install
 
 ## Deployment scenarios affected
 
-The table below provides support details for specific deployment scenarios (Boot Image Version).
+The table below provides support details for specific deployment scenarios (The table represents OS Deployed (vertical) and Boot Image Version(Horizontal) ).
```
Comment (JohanFreelancer9) [inline windows/deployment/wds-boot-support.md:29]:
```suggestion
The table below provides support details for specific deployment scenarios. (The table represents OS Deployed (vertical) and Boot Image Version (horizontal).)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@aczechowski Please merge this PR. Thanks

**Comment by frankroj** [comment]
I am going to take an alternate approach to updating the documentation besides the one proposed. I am going to label the top row as "Boot.wim for" and then label the columns as "OS Being Deployed". This should clarify what the table means.

**Comment by frankroj** [comment]
Fixes https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10973

**Comment by frankroj** [comment]
Fixes #10973

**Comment by frankroj** [comment]
Made suggested changes. Merging and closing PR.

**Pull request merged**